src/components/generic(ui)/create-new-history-container/Create-new-history.tsx

```diff
@@ -0,0 +1,22 @@
+import { CreateNewHistoryBoxProps} from "./Create-new-history-interface";
+import './Create-new-history.css'
+import  plus from '../../../assets/image/icons/plus.svg'
+import Icon from "../icon/Icon";
+import { t } from "i18next";
+import { useTranslation } from "react-i18next";
+;
+
+const CreateNewHistoryBox: React.FC<CreateNewHistoryBoxProps> = (className) => {
+    const {t} = useTranslation('translation')
+    return(
+       <div className="create-new-history__box">
+       <Icon src={plus} className='plus' />
+       <div className="create-new-history__title">
+        <p className="create-new-history__title"> {t('description.createNewHistory.title')}</p>
```

Comment:
if you want use BEM you needed use such a principle for example

div create-new-history -> div create-new-history__box and etc